creation/web_base/glidein_startup.sh

```diff
@@ -2001,6 +2001,14 @@ do
     fi
 done
 
+# adding system information about unprivileged user namespaces to the glidein classad
+glidein_work_dir=$(gconfig_get GLIDEIN_WORK_DIR "$glidein_config")
+. "$glidein_work_dir"/cvmfs_helper_funcs.sh
+unprivileged_user_namespaces=$(has_unpriv_userns)
+gconfig_add "HAS_UNPRIVILEGED_USER_NAMESPACES" "$unprivileged_user_namespaces"
+condor_vars_file=$(gconfig_get CONDOR_VARS_FILE "${glidein_config}" "-i")
+add_condor_vars_line "HAS_UNPRIVILEGED_USER_NAMESPACES" "S" "-" "+" "Y" "Y" "+"
+
```

Comment:
You made the 2 files above executable. This means that glidein_startup.sh will execute them after download. You can publish the variables from there. No need to source again the helper here.

Reply:
Marco confirmed that saying that the above comment was basically suggesting that the lines of code to publish the information can be included in `cvmfs_helper_funcs.sh` since it would have been downloaded and executed. Therefore, my block of code should be something outside of the methods defined in `cvmfs_helper_funcs.sh` such that it gets executed when the file is sourced.